src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/util/DocumentConverter.scala

```diff
@@ -26,7 +26,8 @@ object DocumentConverter {
       digitalAddress = document.get(DIGITAL_ADDRESS.value),
       address = document.get(ADDRESS.value),
       zipCode = document.get(ZIP_CODE.value),
-      origin = document.get(ORIGIN.value)
+      origin = document.get(ORIGIN.value),
+      kind = "Pubbliche Amministrazioni"
```

Comment:
am I missing something, or should this be something like
```suggestion
      kind = document.get(KIND.value)
```